monitoring/collector/collector.go

```diff
@@ -50,14 +50,14 @@ var (
 	)
 )
 
-// PlanetCollector implements the prometheus.Collector interface.
-type PlanetCollector struct {
+// MetricsCollector implements the prometheus.Collector interface.
```

Comment:
Can you add that the metrics collector is a collection of other collector implementations